main.tf

```diff
@@ -55,12 +55,21 @@ resource "google_compute_subnetwork" "subnetwork" {
   ip_cidr_range            = var.subnets[count.index]["subnet_ip"]
   region                   = var.subnets[count.index]["subnet_region"]
   private_ip_google_access = lookup(var.subnets[count.index], "subnet_private_access", "false")
-  enable_flow_logs         = lookup(var.subnets[count.index], "subnet_flow_logs", "false")
   network                  = local.network_self_link
   project                  = var.project_id
   secondary_ip_range       = [for i in range(length(contains(keys(var.secondary_ranges), var.subnets[count.index]["subnet_name"]) == true ? var.secondary_ranges[var.subnets[count.index]["subnet_name"]] : [])) : var.secondary_ranges[var.subnets[count.index]["subnet_name"]][i]]
   description              = lookup(var.subnets[count.index], "description", null)
   depends_on               = [google_compute_network.network]
+
+  dynamic "log_config" {
+    for_each = var.subnet_log_config
+
+    content {
+      aggregation_interval = lookup(log_config.value, "aggregation_interval", null)
```

Comment:
Based on the requested variable change, these expressions can be simplified to just access attributes.